auctioncellrep/auction_cell_rep_test.go

```diff
@@ -424,6 +426,7 @@ var _ = Describe("AuctionCellRep", func() {
 			Expect(healthy).To(BeTrue())
 
 			Expect(state.CellID).To(Equal(cellID))
+			Expect(state.CellIndex).To(Equal(cellIndex))
```

Comment:
test a non-zero value to validate that it is passing it through and not using the default Go value for integer.